README.md

```diff
@@ -6,26 +6,35 @@ Lisk is a next generation crypto-currency and decentralized application platform
 [![Build Status](https://travis-ci.org/LiskHQ/lisk.svg?branch=development)](https://travis-ci.org/LiskHQ/lisk)
 [![Coverage Status](https://coveralls.io/repos/github/LiskHQ/lisk/badge.svg?branch=development)](https://coveralls.io/github/LiskHQ/lisk?branch=development)
 
-**NOTE:** The following information is applicable to: **Ubuntu 14.04 (LTS) - x86_64**.
+**NOTE:** The following information is applicable to: **Ubuntu 16.04 (LTS) - x86_64**.
```

Comment:
Maybe specify Ubuntu 14.04, 16.04 and 16.10 since we have tested all of those versions